### PR TITLE
fix(sgs): km registration fix

### PR DESCRIPTION
**Pull request by davidkef**
# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

## Summary by CodeRabbit

- **New Features**
  - Enhanced vehicle-related forms and queries across templates by adding mileage reading information, improving data tracking and validation.
- **Enhancements**
  - Improved robustness of vehicle mileage fields to handle null values effectively, enhancing data integrity and user experience.
- **Bug Fixes**
  - Resolved issues in vehicle mileage data handling, ensuring accurate validation, and default value settings to prevent submission errors.
- **Documentation**
  - Updated project and client configuration documentation to reflect system changes and new features.

### CODE REVIEW

File: libs/clients/vehicles/project.json [hunk 1]
```diff
@@ -23,9 +23,9 @@
...
-          "cat <<< $(jq '.components.schemas.ProblemDetails.additionalProperties = false' src/clientConfig.json) > src/clientConfig.json",
-          "cat <<< $(jq '.paths.\"/VehicleReportPdf\".get.responses.\"200\".content.\"application/pdf\".schema.format = \"binary\"' src/clientConfig.json) > src/clientConfig.json",
-          "cat <<< $(jq '.paths.\"/OwnershipReportPdf\".get.responses.\"200\".content.\"application/pdf\".schema.format = \"binary\"' src/clientConfig.json) > src/clientConfig.json",
+          "jq '.components.schemas.ProblemDetails.additionalProperties = false' src/clientConfig.json > src/temp.json && mv src/temp.json src/clientConfig.json",
+          "jq '.paths.\"/VehicleReportPdf\".get.responses.\"200\".content.\"application/pdf\".schema.format = \"binary\"' src/clientConfig.json > src/temp.json && mv src/temp.json src/clientConfig.json",
+          "jq '.paths.\"/OwnershipReportPdf\".get.responses.\"200\".content.\"application/pdf\".schema.format = \"binary\"' src/clientConfig.json > src/temp.json && mv src/temp.json src/clientConfig.json",
```
Comment (thordurhhh) [inline libs/clients/vehicles/project.json:28]: Is there any specific reason for these changes?

**Review by thordurhhh — APPROVED** [review]
code owned files lgtm

**Review by berglindoma13 — APPROVED** [review]
LGTM

**Review by Toti91 — APPROVED** [review]

File: libs/clients/vehicles/project.json (hunk 1 above)
Comment (davidkef) [reply libs/clients/vehicles/project.json:28]: It does exactly the same as the one before. But some reason I always have problems running the other one and after this change I have no problem running this along with others. I have made other try running this before without problems.

**Pull request merged**